core/src/main/java/io/questdb/griffin/SqlOptimiser.java

```diff
@@ -4992,7 +5071,7 @@ QueryModel optimise(
         QueryModel rewrittenModel = model;
         try {
             rewrittenModel = bubbleUpOrderByAndLimitFromUnion(rewrittenModel);
-            //extractCorrelatedQueriesAsJoins(rewrittenModel);
+            //extractCorrelatedQueriesAsJoins(model);
```

Comment:
nit: do we need this commented-out code?

Reply:
Removed in latest commit.